### ISSUE TITLE
Add label bot

### ISSUE THREAD

**Issue by sophia-guo** [AdoptOpenJDK/openjdk-tests#2369]
Label bot can add multi labeler for title, body, comments, commit messages, branch, author or files with automated status checks.

https://github.com/fuxingloh/multi-labeler

Similar to https://github.com/AdoptOpenJDK/ci-jenkins-pipelines/pull/76/. Though we can extend the label based on title to body, comments, commit messages, etc.

related with https://github.com/AdoptOpenJDK/openjdk-tests/issues/1766

**Comment by smlambert** [comment]
What types of labels do you envision?  I would only want to introduce this if the bulk of regular contributors thought it will be useful.  I think it would be useful to add more details to the description to propose which labels would be created based on what types of changes.

**Comment by llxia** [comment]
I agree, I think we need more valuable info. By parsing an issue body/comment, if it can decipher the rerun info (i.e., JDK_VERSION, IMPL, PLATFORM, if the issue is intermittent, etc) and add the labels. I think it will help us to rerun the test in auto re-enable https://github.com/AdoptOpenJDK/openjdk-tests/issues/2368

**Comment by sophia-guo** [comment]
Agree, more information will follow up and this is just throw a thought.

**Comment by smlambert** [comment]
Let's either add more specifics on what we would like here, or close the issue until we have clarity.
